solidity/random-beacon/hardhat.config.ts

```diff
@@ -76,12 +76,17 @@ const config: HardhatUserConfig = {
           testConfig.nonStakingAccountsCount +
           testConfig.stakingRolesCount * testConfig.operatorsCount,
       },
-      tags: ["local"],
       // we use higher gas price for tests to obtain more realistic results
       // for gas refund tests than when the default hardhat ~1 gwei gas price is
       // used
       gasPrice: 200000000000, // 200 gwei
       allowUnlimitedContractSize: true,
+      tags: ["allowStubs"],
```

Comment:
This PR needs for testing https://github.com/keep-network/keep-core/pull/3026 but we should probably also update `/ecdsa` with the same tags.

Reply:
ECDSA has to happen in a separate PR as we need to update the dependency to the random beacon there.

https://github.com/keep-network/keep-core/pull/3053